.gitignore

```diff
@@ -5,4 +5,31 @@
 # These are backup files generated by rustfmt
 **/*.rs.bk
 
-.idea
\ No newline at end of file
+.idea
+
+# General
+.DS_Store
+.AppleDouble
```

Comment:
Lots of this ones seem rather arbitrary, never saw occurrence of them on my setup. Do you where are they coming from?

Reply:
See  https://github.com/github/gitignore . My source for gitignore directives.